variables.server.tf

```diff
@@ -11,6 +11,20 @@ variable "administrator_password" {
   sensitive   = true
 }
 
+variable "administrator_password_wo" {
+  type        = string
+  default     = null
+  ephemeral   = true
+  description = "(Optional) The write-only administrator password for the PostgreSQL Flexible Server. Use this if you want to provide a password that is not readable after creation."
```

Comment:
```suggestion
  description = "(Optional) The Password associated with the administrator_login for the PostgreSQL Flexible Server. This can be used to avoid storing the password in state."
```